cmd/appgw-ingress/main.go

```diff
@@ -135,6 +135,8 @@ func main() {
 	}
 
 	// check if application gateway exists/have get access
+	// When an AAD SP doesn't have access to the app gateway resource group, then AGIC can't read it's role assignments to look for the needed permission.
```

Comment:
Should it be AAD User-assigned Managed Identity instead of Service Principal here?